doc/md/guides/orms/sqlalchemy.md

```diff
@@ -0,0 +1,262 @@
+---
+id: sqlalchemy
+title: Automatic migration planning for SQLAlchemy
+slug: /guides/orms/sqlalchemy
+---
+
+## TL;DR
+* [SQLAlchemy](https://www.sqlalchemy.org) is a Python SQL toolkit and Object Relational Mapper.
+* [Atlas](https://atlasgo.io) is an open-source tool for inspecting, planning, linting and
+  executing schema changes to your database.
+* Developers using SQLAlchemy can use Atlas to automatically plan schema migrations
+  for them, based on the desired state of their schema instead of crafting them by hand.
+
+## Automatic migration planning for SQLAlchemy
+
+SQLAlchemy is a popular ORM toolkit widely used in the Python community. SQLAlchemy allows users to
+manage their database schemas using its [declarative-mapping](https://docs.sqlalchemy.org/en/20/orm/declarative_tables.html)
+feature, which is usually sufficient during development and in many simple cases.
+
+However, at some point, teams need more control and decide to employ
+the [versioned migrations](/concepts/declarative-vs-versioned#versioned-migrations) methodology,
+which is a more robust way to manage a database schema.
+
+The native way to manage migrations with SQLAlchemy is to use the [Alembic](https://alembic.sqlalchemy.org/en/latest/) migration tool.
+Alembic can [automatically generate](https://alembic.sqlalchemy.org/en/latest/autogenerate.html#auto-generating-migrations)
+migration scripts from the difference between the current state of the database and the desired state of the application.
+In the context of versioned migrations, the current state can be thought of as the database schema that would have been
+created by applying all previous migration scripts.
```

Comment:
```suggestion
```
Correct me if I'm wrong, but this isn't true for alembic where the comparison is don't between a live db that's supposed to have the current schema and the desired state.